src/app/shared/helpers/config.ts

```diff
@@ -1,7 +1,7 @@
 import { environment } from 'src/environments/environment';
 
 interface TokenConfig { authType: 'tokens' }
-type CookieConfig = { authType: 'cookies' } & ({ secure: true, sameSite: boolean } | { secure: boolean, sameSite: true });
+interface CookieConfig { authType: 'cookies', secure: boolean }
```

Comment:
Hum... this `secure` is just whether the backend is in https or not. 
We should probably move `authType` to `Environment` directly and compute `cookie_secure` in code.

Can be done in another PR.